File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkCommand.java
package com.matter.controller.commands.pairing;

import chip.devicecontroller.ChipDeviceController;
import com.matter.controller.commands.common.CredentialsIssuer;

public final class PairOnNetworkCommand extends PairingCommand {
  public PairOnNetworkCommand(ChipDeviceController controller, CredentialsIssuer credsIssue) {
    super(controller, "onnetwork", PairingModeType.ON_NETWORK, PairingNetworkType.NONE, credsIssue);
  }

  @Override
  protected void runCommand() {}
}

File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkCommand.kt
package com.matter.controller.commands.pairing

import chip.devicecontroller.ChipDeviceController
import com.matter.controller.commands.common.CredentialsIssuer

class PairOnNetworkCommand(controller: ChipDeviceController, credsIssue: CredentialsIssuer?) :
  PairingCommand(controller, "onnetwork", PairingModeType.ON_NETWORK, PairingNetworkType.NONE, credsIssue) {
  override fun runCommand() {}
}